app/src/main/java/com/winlator/contents/ContentProfile.java

```diff
@@ -16,6 +16,10 @@ public class ContentProfile {
     public static final String MARK_WINE_BINPATH = "binPath";
     public static final String MARK_WINE_LIBPATH = "libPath";
     public static final String MARK_WINE_PREFIX_PACK = "prefixPack";
+    public static final String MARK_PROTON = "proton";
+    public static final String MARK_PROTON_BINPATH = "binPath";
+    public static final String MARK_PROTON_LIBPATH = "libPath";
+    public static final String MARK_PROTON_PREFIX_PACK = "prefixPack";
```

Comment:
Need for defining these for custom protons when it needs to check the profile.json to find the correct path.